src/eigen.jl

```diff
@@ -79,8 +79,9 @@ end
     H1 = Hessenberg{typeof(zero(eltype(S.data))),typeof(T),typeof(S.data),typeof(tau),typeof(false)}(T, 'L', S.data, tau, false)
     A = S.data
     H = SymTridiagonal(zeros(eltype(E), n), E)
-    trisol = eigen!(H)
-
+    shift = norm(H)
+    trisol = eigen!(H.*shift)
```

Comment:
This doesn't make sense — by "shift" I was thinking about `H + shift*I`, which breaks the degeneracies in |λ| between the ± imaginary parts.

If you do a multiplicative scaling, it's not a shift — it doesn't do anything about the degeneracies.

Reply:
I tried to shift before trying to scale but it didn't work. I will try to find another mathematically interpretable and logical solution.